scene/gui/file_dialog.cpp

```diff
@@ -1043,6 +1047,7 @@ void FileDialog::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "access", PROPERTY_HINT_ENUM, "Resources,User Data,File System"), "set_access", "get_access");
 	ADD_PROPERTY(PropertyInfo(Variant::STRING, "root_subfolder"), "set_root_subfolder", "get_root_subfolder");
 	ADD_PROPERTY(PropertyInfo(Variant::PACKED_STRING_ARRAY, "filters"), "set_filters", "get_filters");
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "case_sensitive_filter", PROPERTY_HINT_ENUM, "Sensitive,Insensitive,OS Default"), "set_case_sensitive_filter", "get_case_sensitive_filter");
```

Comment:
```suggestion
	ADD_PROPERTY(PropertyInfo(Variant::INT, "case_sensitive_filter", PROPERTY_HINT_ENUM, "Case Sensitive,Case Insensitive,OS Default"), "set_case_sensitive_filter", "get_case_sensitive_filter");
```
I'd say this is clearer

Reply:
Or, by some standards:
```suggestion
	ADD_PROPERTY(PropertyInfo(Variant::INT, "case_sensitive_filter", PROPERTY_HINT_ENUM, "Case-Sensitive,Case-Insensitive,OS Default"), "set_case_sensitive_filter", "get_case_sensitive_filter");
```